examples/category-results-materialize.html

```diff
@@ -24,7 +24,12 @@
           source="demo.sourceFn(query)">
 
         <ngc-omnibox-selections class="chips">
-          <div class="chip">{{selection.make_display}}</div>
+          <div class="chip">
+            {{selection.make_display}}
+            <i class="close material-icons" ng-click="omnibox.removeFromSelections(selection)">
```

Comment:
Yea, but a lot of libraries use it to mean "icon". This is the markup that materialize uses.